packages/error-reporting/package.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
@stephenplusplus I have been trying to figure out the same thing myself :)
I figure, at this point, we should do whatever impacts your workflow the least -- if that means exclusion from top-level runs I can take a whack at modifying the top-level package.json. Otherwise I don't have any issues being included in the top-level test runs. wdyt?